packages/documentation/utilities/date.ts

```diff
@@ -20,4 +21,4 @@ export const today = (templateFormat: string = ISO8601): string =>
 export const getLast = (
 	unit: ManipulateType,
 	templateFormat: string = ISO8601,
-) => dayjs().subtract(1, unit).format(templateFormat)
+): string => dayjs().subtract(1, unit).format(templateFormat)
```

Comment:
added because the other funcction in this file was also explicit about the return (consistency)